stored_queries/ncbi_tax/ncbi_taxon_search_sci_name.yaml

```diff
@@ -0,0 +1,21 @@
+# Search for an ncbi taxon with a scientific name
+# Results are limited to 10k
+name: ncbi_taxon_search_sci_name
+params:
+  type: object
+  required: [search_text]
+  properties:
+    search_text:
+      type: string
+      title: Search text
+      description: Text to search on for the scientific name
+    offset:
+      type: integer
+      default: 0
+    limit:
+      type: integer
+      default: 20
+query: |
+  FOR doc in FULLTEXT(ncbi_taxon, "scientific_name", @search_text)
+    LIMIT @offset, @limit
```

Comment:
Doesn't seem like these are tested

Reply:
Added a test for the max offset. Not going to other pagination results for now